src/main/java/com/google/devtools/build/lib/packages/AttributeMap.java

```diff
@@ -57,6 +57,8 @@ default String describeRule() {
    *
    * <p>If the rule doesn't have this attribute with the specified type, throws an
    * {@link IllegalArgumentException}.
+   * 
+   * Note that null may be the default value, check if default set
```

Comment:
TODO Rework doc

Reply:
Actually, not completely. It refers to `Type#getDefaultValue` however this is handled by the `Attribute` definition itself (has `valueSet ? value : type.getDefaultValue(),`).